eng/tools/dependency-testing/index.js

```diff
@@ -49,34 +49,45 @@ function outputTestPath(projectFolderPath, sourceDir, testFolder) {
   console.log(`Emitted variable "PackageTestPath" with content: ${testPath}`)
 }
 
+async function updateScripts(testPackageJson, packageJsonContents){
```

Comment:
A comment describing why this function is valuable will help us in the future when we come back to this.

Reply:
done